src/main/java/com/google/devtools/build/lib/starlarkbuildapi/python/PyRuntimeInfoApi.java

```diff
@@ -46,6 +47,7 @@
 public interface PyRuntimeInfoApi<FileT extends FileApi> extends StarlarkValue {
 
   static final String DEFAULT_STUB_SHEBANG = "#!/usr/bin/env python3";
+  static final String DEFAULT_BOOTSTRAP_TEMPLATE = "//tools/python:python_bootstrap_template.txt";
```

Comment:
Should this have "@bazel_tools"? Or is that implicit in this context for some reason?

Reply:
Actually added it back, it's used (but via the subclass PyRuntimeInfo, that's why I didn't find it via `rg`). I added a comment that it should be used with `getToolsLabel()` to use it correctly